### ISSUE TITLE
[BUG] openshift local never fully starts before VM becomes unresponsive

### ISSUE THREAD

**Issue by twhite0** [crc-org/crc#3480]
### General information

  * OS: macOS 
  * Hypervisor: vfkit
  * Did you run `crc setup` before starting it (Yes)?
  * Running CRC on: Laptop 

## CRC version
```bash
CRC version: 2.12.0+ea98bb41
OpenShift version: 4.11.18
Podman version: 4.2.0
```
  
## CRC status
```bash
DEBU CRC version: 2.12.0+ea98bb41
DEBU OpenShift version: 4.11.18
DEBU Podman version: 4.2.0
DEBU Running 'crc status'
CRC VM:          Running
OpenShift:       Starting (v4.11.18)
RAM Usage:       5.631GB of 12.52GB
Disk Usage:      12.74GB of 53.14GB (Inside the CRC VM)
Cache Usage:     36.8GB
Cache Directory: /Users/twhite0/.crc/cache
```

## CRC config
```bash
- consent-telemetry                     : no
- cpus                                  : 6
- disk-size                             : 50
- http-proxy                            : http://127.0.0.1:8888
- https-proxy                           : http://127.0.0.1:8888
- memory                                : 12288
- no-proxy                              : localhost,127.0.0.1,.testing
- proxy-ca-file                         : /Users/twhite0/git/hybrid-registry-poc/proxy.pem
```

Running behind a corporate proxy.  proxy-ca-file has been set but does not seem to be taking hold.  https://github.com/crc-org/crc/issues/3479

## Host Operating System
```bash
roductName:	macOS
ProductVersion:	12.6.2
BuildVersion:	21G320
```

### Steps to reproduce

  1. crc delete
  2. crc setup
  3. crc start

### Expected

A running and accessible OpenShift local.

### Actual

* After 3-5 minutes the VM becomes unresponsive
* Connection attempts on 2222 get connection resets.
* `crc stop` followed by `crc start` do no work...the VM seems to be unusable.
* ssh'ing into vm becomes unresponsive.

### Logs
https://gist.github.com/twhite0/30f059e6bd1ad8417ac0cca5fe805b90

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
```
DEBU openshift-apiserver operator is degraded, Reason: APIServerDeployment_UnavailablePod
DEBU openshift-apiserver operator not available, Reason: APIServerDeployment_NoPod
DEBU openshift-controller-manager operator is still progressing, Reason: _DesiredStateNotYetAchieved
DEBU openshift-controller-manager operator not available, Reason: _NoPodsAvailable
DEBU operator-lifecycle-manager-packageserver operator is still progressing, Reason:
INFO 5 operators are progressing: authentication, console, image-registry, openshift-controller-manager, operator-lifecycle-manager-packageserver
DEBU authentication operator is still progressing, Reason: OAuthServerDeployment_NewGeneration
DEBU authentication operator not available, Reason: OAuthServerDeployment_NoPod
DEBU console operator not available, Reason: RouteHealth_RouteNotAdmitted
DEBU image-registry operator not available, Reason: NoReplicasAvailable
DEBU image-registry operator is still progressing, Reason: DeploymentNotCompleted
DEBU kube-apiserver operator is still progressing, Reason: NodeInstaller
DEBU kube-controller-manager operator is still progressing, Reason: NodeInstaller
DEBU node-tuning operator not available, Reason: TunedUnavailable
DEBU openshift-apiserver operator not available, Reason: APIServerDeployment_NoPod::APIServices_PreconditionNotReady
```

API Server remains in a degraded state.

**Comment by praveenkumar** [comment]
```
 After 3-5 minutes the VM becomes unresponsive
 Connection attempts on 2222 get connection resets.
 crc stop followed by crc start do no work...the VM seems to be unusable.
 ssh'ing into vm becomes unresponsive.
```

Looking at it looks like `crc daemon` is responding can you check if daemon process is running?

```
$ ps axu | grep crc
```

Also check the logs `$HOME/.crc/crcd.log`

**Comment by twhite0** [comment]
@praveenkumar:  Thanks for engaging in my issue, appreciate your time.  I updated my [gist](https://gist.github.com/twhite0/30f059e6bd1ad8417ac0cca5fe805b90) with the daemon logs.  

the following error coincides with the `crc start` and ssh's (if connected) becoming unresponsive.

`ERRO cannot receive packets from , disconnecting: cannot read size from socket: EOF`

**Comment by praveenkumar** [comment]
@twhite0  Can you please try latest version of CRC where we fixed some proxy related bugs and create new issue if it is still exist?
